mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_organization_notification_config_test.go

```diff
@@ -0,0 +1,79 @@
+package securitycenterv2_test
+
+import (
+    "testing"
+
+    "github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+    "github.com/hashicorp/terraform-provider-google/google/acctest"
+    "github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccSecurityCenterV2OrganizationNotificationConfig_basic(t *testing.T) {
+    t.Parallel()
+
+    context := map[string]interface{}{
+        "org_id":        envvar.GetTestOrgFromEnv(t),
+        "random_suffix": acctest.RandString(t, 10),
+    }
+
+    acctest.VcrTest(t, resource.TestCase{
+        PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+        ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+        Steps: []resource.TestStep{
+            {
+                Config: testAccSecurityCenterV2OrganizationNotificationConfig_basic(context),
+            },
+            {
+                ResourceName:      "google_scc_v2_organization_notification_config.default",
+                ImportState:       true,
+                ImportStateVerify: true,
+                ImportStateVerifyIgnore: []string{
+                    "config_id",
+                },
+            },
+            {
+                Config: testAccSecurityCenterV2OrganizationNotificationConfig_update(context),
+            },
+        },
+    })
+}
+
+func testAccSecurityCenterV2OrganizationNotificationConfig_basic(context map[string]interface{}) string {
+    return acctest.Nprintf(`
```

Comment:
These `Nprintf` statements still won't work. Nprintf does not use `%s` like the regular `Printf` functions. Instead, you pass in `context` and reference fields by name.

Please review both of these functions (here and below), and make sure that they are correct. You can search the rest of the codebase for `Nprintf` for other examples of how it should be used.

Reply:
addressed